kubernetes-repositories.md

```diff
@@ -0,0 +1,106 @@
+## Kubernetes Repositories
+
+
+This document attempts to describe a process for creating and associating github repositories with the Kubernetes project.
+
+The document presents a tiered system of repositories with increasingly strict requirements in an attempt to provide the right level of oversight and flexibility for a variety of different projects.
+
+
+### Associated Repositories
+
+Associated repositories conform to the Kubernetes community standards for a repository, but otherwise have no restrictions. Associated repositories exist solely for the purpose of making it easier for the Kubernetes community to work together. There is no implication of support or endorsement of any kind by the Kubernetes project, the goals are purely logistical. 
```

Comment:
it should specify somewhere if those repos are brand new or can be existing repos that are migrated to the "kubernetes" org at large.

Reply:
I think that this applies to both types of repositories, both new ones and ones that might be migrated out of incubator.

But we're not going to try to modify the status quo, we won't force repositories to move...